clients/nasaClient.ts

```
@@ -0,0 +1,18 @@
+const apiKey: string | undefined = process.env.NASA_API_KEY
+export default class nasaClient {
+
+    static getRoverList(): Promise<void | Rover[]> {
+        const apiUrl: string = `https://api.nasa.gov/mars-photos/api/v1/rovers?api_key=${apiKey}`
+        return axios.get(apiUrl)
+            .then((response: any): Rover[] => response.data.rovers)
+            .catch((error: any): void => {
+                console.error(error)
+            })
+    }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)